bgpd/bgp_vty.c

```diff
@@ -1368,6 +1368,23 @@ DEFPY (no_bgp_router_id,
 	return CMD_SUCCESS;
 }
 
+DEFPY (bgp_suppress_fib_pending,
+       bgp_suppress_fib_pending_cmd,
+       "[no] bgp suppress-fib-pending",
+       NO_STR
+       BGP_STR
+       "Advertise only routes that are programmed in kernel to peers\n")
+{
+	VTY_DECLVAR_CONTEXT(bgp, bgp);
+	int idx = 0;
+	bool yes = true;
+
+	if (argv_find(argv, argc, "no", &idx))
+		yes = false;
```

Comment:
This is a defpy, `no` is already a variable. Use `!no` in place of `yes` and delete 1379 - 1384.

Reply:
changes done